interpolate/src/commonMain/kotlin/LinearInterpolator.kt

```diff
@@ -24,7 +16,7 @@ private class LinearIntInterpolator(
     override fun invert(value: Int): Float = (value - start).toFloat() / range
 }
 
-private class LinearFloatInterpolator(
+internal class LinearFloatInterpolator(
```

Comment:
Why did the visibility need to change for Kotlinter?

Reply:
Kotlinter wanted me to rename the file to match the public functions at the top of it, even though they weren't really the _meat_ of the file **and** another file with that name already existed. So I moved those functions to the other, pre-existing file. This satisfied Kotlinter, but the visibility here needed to change to let me move those functions.